webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/users/UserRepository.scala

```diff
@@ -616,10 +619,16 @@ class JdbcUserRepository(doobie: Doobie) extends UserRepository {
 
     transactIOResult(s"Error when saving session '${sessionId.value}' info for user '${userId}'")(xa => {
       (for {
-        _ <- session.update.run
-        _ <- lastLogin.update.run
-      } yield ()).transact(xa)
-    }).unit
+        _   <- lastLogin.update.run
+        err <- session.update.run
+                 .as(none[String])
+                 .onUniqueViolation(s"User session for ${userId} has already been saved".some.pure[ConnectionIO])
+      } yield err).transact(xa)
+    }).flatMap {
+      case None      => ZIO.unit
+      // it should not be fatal if a login already happened, but it could have security implications
+      case Some(err) => ApplicationLoggerPure.warn(err)
```

Comment:
I don't think we want that. 
The warning here will be non actionnable for users, but still source of worry for people who monitor rudder logs and escalate on warning and errors. Especially if they found the comment `but it could have security implications` - they will want to know exactly which, and how to prevent them. 

Would it be possible to force the user to completely reautent in the case of a stalled sessionid ? IIUC, that happen only when the browser kept a session cookie that should have been destroyed because Rudder lost that context (restarted or anything). 
So can we force a full reauth (even if it means calling "user.logout" on sql error here) and put a login error message like "Rudder authentication service restarted during your login. Plean log again" ?